src/remote/cozy.js

```diff
@@ -0,0 +1,25 @@
+import { Cozy as CozyClient } from 'cozy-client-js'
+import fetch from 'node-fetch'
```

Comment:
As `cozy-client-js` makes heavy use of the `fetch` API, and the goal was to use it in `cozy-desktop`, it makes sense to use the same HTTP request API everywhere. It may also make it easier to extract some code to the lib when it makes sense.